torch/csrc/jit/codegen/cuda/lower_validation.cpp

```diff
@@ -109,16 +181,20 @@ class VectorizeValidator : public OptInDispatch {
     } else {
       vectorized_id_ = m->inner();
     }
+    domains_.insert(m->outer());
+    domains_.insert(m->inner());
   }
 
  private:
+  std::unordered_set<IterDomain*> domains_;
```

Comment:
What is this member? Domains of what?

Reply:
This is all the iter_domains associated with the vectorized iter_domain. I then filter all the iter_domains to get only the root domains.

Reply:
I think you can do `IterVsitor::getInputsTo(vectorized_id)`, which should return only root IDs that are an input to `vectorized_id`.

Reply:
It's traversing the expression list anyway, so it may actually make sense not to do additional traversal with `getInputsTo`. In any case, adding `s->outer()` and `s->inner()` in the split case should not be necessary, right?

Reply:
Why isn't adding either necessary?

Reply:
If I understand it correctly, it is intended to find root axes by traversing backwards, right? The outer and inner of a `split` are output, so why are they needed to be added? It doesn't make the end result wrong as only root axes are used from the final resulting set, but I don't understand why they need to be added.